api/environments/identities/tests/test_models.py

```diff
@@ -667,11 +672,18 @@ def test_generate_traits_with_persistence(self):
         # and the database matches it
         assert Trait.objects.filter(identity=identity).count() == 3
 
-    def test_generate_traits_without_persistence(self):
+        # and put_item was called with correct args
+        identity_dict = build_identity_dict(identity)
+        dynamo_identity_table.put_item.assert_called_with(Item=identity_dict)
+
+    @mock.patch("environments.identities.models.dynamo_identity_table")
+    def test_generate_traits_without_persistence(self, dynamo_identity_table):
         # Given
         identity = Identity.objects.create(
             identifier="identifier", environment=self.environment
         )
+
+        dynamo_identity_table.put_item = mock.MagicMock()
```

Comment:
As before, I don't think you need this - an attribute of a MagicMock is already a MagicMock itself